lib/dfe/analytics/services/postgres_checksum_calculator.rb

```diff
@@ -0,0 +1,56 @@
+require_relative '../shared/service_pattern'
```

Comment:
Maybe a simple collaboration spec to check the connection is executed with the expected SQL statement ?

Reply:
Now added 👍